### PR TITLE
CP-21662:replace the icon for new vm template(asianux and turbo linux)in xencenter.

### PR DESCRIPTION
**Pull request by minli1**
Replace the icon for new guest vm: asianux and turbo linux.
Had been tested manually and CI.

### CODE REVIEW

File: XenModel/XenAPI-Extensions/VM.cs [hunk 1]
```diff
@@ -1022,17 +1022,19 @@ public enum VmTemplateType
...
-            SciLinux = 9,
-            Suse = 10,
-            Ubuntu = 11,
-            Citrix = 12,
-            Solaris = 13,
-            Misc = 14,
-            Snapshot = 15,
-            SnapshotFromVmpp = 16,
-            Count = 17  //bump this if values are added
+            Asianux  = 7,
```
Comment (MihaelaStoica) [inline XenModel/XenAPI-Extensions/VM.cs]: I think the rule here is to sort the VMs that belong to the same group (E.g. Linux VMs) in alphabetical order.

Comment (Zhengchai) [inline XenModel/XenAPI-Extensions/VM.cs]: can we just make this codes onto feature branch at the moment before we submit the PR to master. thanks.

Comment (minli1) [inline XenModel/XenAPI-Extensions/VM.cs]: Hi,Mihaela. I had changed the icon with a grey border and also the order of linux vms. 
Please review it.  Thanks a lot.

Comment (minli1) [inline XenModel/XenAPI-Extensions/VM.cs]: Hi,Mihaela. I had change the icons with grey border and also the order of linux vms.
Please review and comment. Thanks a lot.

**Comment by MihaelaStoica** [comment]
Looks good now. Merging.

**Comment by MihaelaStoica** [comment]
Please note that we have renamed branch REQ-440 to REQ-398

**Pull request merged**